spec/timezone.html

```diff
@@ -380,6 +382,72 @@ <h1>Properties of Temporal.TimeZone Instances</h1>
   <emu-clause id="sec-temporal-timezone-abstract-ops">
     <h1>Abstract operations</h1>
 
+    <emu-clause id="sec-temporal-time-zone-records">
+      <h1>Time Zone Records</h1>
```

Comment:
I worry a bit that this name is too generic for the fairly narrow use case of caching methods of a Temporal object. There are many other possible reasons why you might have something called a "Time Zone Record", e.g. for doing identifier canonicalization or something in 402 related to formatting. I'm not saying that those should be called "Time Zone Record" either, only that claiming the general name for a specific use case doesn't seem like a good idea because it might be confused with other use cases down the line.

So maybe a better name would be something like "Temporal Time Zone Methods Record"?

Same feedback applies to the calendar record type.

Reply:
Revising this now. "Temporal Time Zone Methods Record" is quite a mouthful ... I'll go with Time Zone Methods Record for now, shout if you disagree.